backend/database/migrations/2021_10_25_061828_create_user_learn_words_table.php

```diff
@@ -15,7 +15,7 @@ public function up()
     {
         Schema::create('user_learn_words', function (Blueprint $table) {
             $table->id();
-            $table->foreignId('choice_id')->constrained()->onDelete('cascade');
+            $table->foreignId('question_choice_id')->constrained()->onDelete('cascade');
```

Comment:
make new migration file to change the foreignId, because with this you will always run php artisan migrate:fresh and it is not good for the production. 

Drop the foreign key then add it again in the new migration and run migrate.